setup.py

```diff
@@ -2,51 +2,55 @@
 from setuptools import setup
 
 __author__ = "Martin Uhrin"
-__license__ = "GPLv3 and MIT, see LICENSE file"
-__contributors__ = "Sebastiaan Huber"
+__license__ = "GPLv3 and MIT"
+__contributors__ = "Sebastiaan P. Huber"
 
 about = {}
 with open('kiwipy/version.py') as f:
     exec(f.read(), about)
 
-setup(name="kiwipy",
```

Comment:
Did you want to include these changes in this PR or rather in the misc/docs one?

Reply:
I'd rather just leave it now because it might make my life easier when I rebase the docs PR.

Reply:
It should not cause a fuss, since presumably this is the state that your docs PR is building off, but again I won't hold your feet to the fire ;)